src/core/utilities.js

```diff
@@ -100,22 +100,23 @@ export function objectType( obj ) {
 		type = match && match[ 1 ];
 
 	switch ( type ) {
-		case "Number":
-			if ( isNaN( obj ) ) {
-				return "nan";
-			}
-			return "number";
-		case "String":
-		case "Boolean":
-		case "Array":
-		case "Set":
-		case "Map":
-		case "Date":
-		case "RegExp":
-		case "Function":
-		case "Symbol":
-			return type.toLowerCase();
+	case "Number":
```

Comment:
Switch statement cases should be aligned with the switch keyword and everything else goes from there. See ["Switch" statements](https://contribute.jquery.org/style-guide/js/#switch-statements) section of the jQuery style guide.